redfish-core/lib/service_root.hpp

```diff
@@ -20,12 +20,22 @@
 #include <registries/privilege_registry.hpp>
 #include <utils/systemd_utils.hpp>
 
+#include <filesystem>
```

Comment:
Is this include required here?

Reply:
I think ofstream is provided by filesystem.

Reply:
but where is ofstream in this file?